dftimewolf/lib/collectors/gcloud.py

```diff
@@ -59,11 +59,13 @@ def Process(self):
       new_disk = gcp_forensics.CreateDiskCopy(
           self.remote_project.project_id,
           self.analysis_project.project_id,
+          None,
```

Comment:
This might be some weird merge artifact, but i removed the extraneous `None` which has disappeared from the latest version of LCF: https://github.com/google/cloud-forensics-utils/blob/e6e53f8bbf2eec86e6e4f4a54bcddf6ead7902d3/libcloudforensics/providers/gcp/forensics.py#L34

Reply:
Same thing in the tests for this function

Reply:
Done